### ISSUE TITLE
Install grunt and grunt-cli versions as specified in the npm package.json

### ISSUE THREAD

**Issue by tjsingleton** [srs/gradle-grunt-plugin#32]
Given a package.json exists, when the installGrunt task is executed then install the versions specified in the package.json.

**Comment by tjsingleton** [comment]
@srs Our builds started failing today because of an issue with the version of grunt-cli being used. I was about to work around it by doing this: `installGrunt.setArgs( ['install', 'grunt-cli@1.1.0', 'grunt@~0.4.2'] )`. 

I'd be happy to implement support for both manually configuring the versions, as well as loading it from a package.json if it exists and is applicable. Would you be open to a PR?

**Comment by srs** [comment]
Hi.

Always open for PR's.
tir. 5. apr. 2016 kl. 03.53 skrev TJ Singleton <notifications@github.com>:

**Comment by tspaeth** [comment]
Yes, I can confirm this issue. Also started to happen here as the grunt-cli always is installed in the most recent version and that seems to have some issues for whatever reason.

**Comment by tspaeth** [comment]
Issue is buried down in the grunt update to 1.0.0 - https://github.com/gruntjs/grunt/issues/1499 & https://github.com/gruntjs/grunt/issues/1491

**Comment by Neil201** [comment]
I would like to see this implemented also, but I think this is the same request as issue #19 here - https://github.com/srs/gradle-grunt-plugin/issues/19

**Comment by tjsingleton** [comment]
I read #19 as setting the gradle project version.

**Comment by Neil201** [comment]
yeah sorry your right , my bad.

**Comment by stevematyas** [comment]
+1

**Comment by eekboom** [comment]
Workaround for me:
* Do not use this plugin's gruntInstall task
* Instead
   * Add grunt-cli to package.json
   * Instead use npmInstall of the gradle npm plugin

**Comment by srs** [comment]
Anyone has a pull request for this?
